src/app/services/auth/auth.service.ts

```diff
@@ -81,8 +81,20 @@ export class AuthService {
     return notExpired;
   }
 
-
   signOut() {
+
     window.location.href = '/api/logout';
   }
+
+  logOutAndRedirect() {
+
+    console.log('logOutAndRedirect');
+    const { hostname, port } = window.location;
+    const portNumber = port ? `:${port}` : '';
+
+    const baseUrl = `http://${hostname}${portNumber}`;

```

Comment:
please don't do this, let the backend do it? also protocol will be wrong etc, not always http

Reply:
Awaiting the Open Id logout route, then we will refactor this as discussed with Paddy.